caracal/sample_configurations/workflow_new.yml

```diff
@@ -51,7 +51,7 @@ flag:
     enable: false
     scans: '0'
   flag_mask:
-    enable: true
+    enable: false
     mask: labelled_rfimask.pickle.npy
```

Comment:
@paoloserra , the flag_mask action is being disabled because this is not a MeerKAT specific configuration. In keeping with that idea, could you please remove the name of the mask (that's MeerKAT specific too). Note that the schema no longer has a default for this, so the user would need specifically select the appropriate file.

Reply:
done